Sources/MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -68,10 +68,19 @@ open class CarPlayNavigationViewController: UIViewController {
      */
     public private(set) var styleManager: StyleManager?
     
+    /**
+     Allows to control highlighting of the destination building on arrival. By default destination buildings will not be highlighted.
+     */
+    public var waypointStyle: WaypointStyle = .annotation
+    
     var mapTemplate: CPMapTemplate
     var carInterfaceController: CPInterfaceController
     
     private var isTraversingTunnel = false
+    private var approachingDestinationThreshold: CLLocationDistance = 250.0
```

Comment:
I think this is a copy from `NavigationViewController` and we shall think how to restructure code so that there won't be so much duplicated code.

At very least, 250.0 is a shared constant that can be extracted.

Reply:
Some refactoring was made and now building highlighting logic is the same for `CarPlayNavigationViewController` and `NavigationViewController`.